index.html

```diff
@@ -187,6 +187,37 @@ <h4>Skills</h4>
 </section>
 
 
+    <form class="last_section" action="https://formspree.io/f/xbjwkevl" method="post">
+        <div class="logo1">
+        </div>
+        <div class="logo_container">
+            <div class="logo2">
+            </div>
+        
+            <div class="logo3">
+            </div>
+            
+    </div>
+    <div>
+        <h1 class="contact_me">Contact me</h1>
+    </div>
+ 
+        <p class="last_para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <div>
+   <input type="text" placeholder="Enter your name" maxlength="30" required class="text1"> 
+</div>
+   <input type="email" placeholder="Enter your Email" required class="text1">
+   <div>
+   <textarea class="textarea_form" rows="4" maxlength="500" placeholder="Write your message here" cols="50"></textarea>
```

Comment:
- [ ] On submission, `formspree` in rendering an error. Please make sure add `name` attributes to address this issue.
<img width="336" alt="Screenshot 2022-03-23 at 12 41 14" src="https://user-images.githubusercontent.com/77038610/159658224-5dee3ed5-1499-430b-b96e-0e2dcc4bbd70.png">
After adding name` attributes please make sure that form works as expected (formspree successfully sends emails)